tokio-quiche/Cargo.toml

```diff
@@ -0,0 +1,74 @@
+[package]
+name = "tokio-quiche"
+version = "0.1.0"
+repository = { workspace = true }
+description = "Asynchronous wrapper around quiche"
+keywords = ["quic", "http3", "tokio"]
+categories = { workspace = true }
+rust-version = "1.83"
+edition = { workspace = true }
+readme = "README.md"
+
+[features]
+# Forwarded quiche features for re-exports
+fuzzing = ["quiche-cf/fuzzing", "quiche-mallard?/fuzzing"]
+quiche_internal = ["quiche-cf/internal", "quiche-mallard?/internal"]
+
+# Enable extra timing instrumentation for QUIC handshakes, including protocol
+# overhead and network delays.
+perf-quic-listener-metrics = []
+
+# Enable raw public key (RPK) support for QUIC handshakes.
+rpk = ["boring/rpk"]
+
+# Use the quiche-mallard fork in place of upstream quiche.
+# quiche-mallard replaces quiche's original congestion control
+# implementation with one adapted from google/quiche.
+gcongestion = ["dep:quiche-mallard"]
+# Use quiche-mallard with zero-copy send calls.
+zero-copy = ["gcongestion"]
+
+# Enable SSLKEYLOGFILE capturing for QUIC connections.
+capture_keylogs = []
+
+# Enable scheduling & poll duration histograms for tokio tasks.
+tokio-task-metrics = []
+
+[dependencies]
+boring = { workspace = true }
+buffer-pool = { path = "../buffer-pool" }
```

Comment:
you can't publish tokio-quiche without specifying a version for the helper crates (crates.io won't allow a path-only dependency)

Reply:
I know, but I have to publish those first before I can change it here. I'm going to do that in a follow up PR

Reply:
cargo should prefer the path over `version`+crates.io. that's how we use it in other workspaces as well.